airbnb/src/components/Card.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export default function Card(props) {
+  let badgeText;
+  if (props.openSpots === 0) {
+    badgeText = "SOLD OUT";
+  } else if (props.location === "Online") {
+    badgeText = "ONLINE";
+  }
+
+  return (
+    <div className="card">
+      {badgeText && <div className="card--badge">{badgeText}</div>}
+
+      <img src={props.coverImg} alt="atheletePicture" className="card--image" />
+
+      <div className="card--stats">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <title>star</title>
+          <path d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z" />
+        </svg>
```

Comment:
SVGs need to be in a separate file
Either  `.svg` & use with `<img>`
Or use as a React component

Reply:
Ok ma'am